check-ssl-cert/lib/check-ssl-cert_test.go

```diff
@@ -0,0 +1,172 @@
+package checksslcert
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"net"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"testing"
+
+	"github.com/mackerelio/checkers"
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	cert_pem   = "cert.pem"
+	cert_key   = "key.pem"
+	client_crt = "client.crt"
+	client_key = "client.key"
+)
+
+// generate "cert.pem", "key.pem", "client.crt", "client.key" in tmpDir
```

Comment:
What did you think to generate locally those keys and certificates and to commit them into testdata/ directory?

Reply:
expiration.